content/docs/1.18/opentelemetry.md

```diff
@@ -112,4 +112,4 @@ spec:
 
 <3> Health check port has to match Jaeger component health check port from the [deployment page](deployment.md).
 
-Once Jaeger OpenTelemetry binaries are released in a stable stream the Jaeger Operator will automatically use new set of images and properly change readiness probes to the new ports.
\ No newline at end of file
+Once Jaeger OpenTelemetry binaries are released in a stable stream the Jaeger Operator will automatically use new set of images and properly change readiness probes to the new ports.
```

Comment:
Not sure what has changed on this line? Although could update "use new set" -> "use a new set"

Reply:
updated